pkgs/top-level/all-packages.nix

```
@@ -168,6 +168,8 @@ in
   buildFHSUserEnvChroot = callPackage ../build-support/build-fhs-userenv { };
   buildFHSUserEnvBubblewrap = callPackage ../build-support/build-fhs-userenv-bubblewrap { };
 
+  grakn = callPackage ../servers/grakn/grakn.nix {};
```

```suggestion
  grakn = callPackage ../servers/grakn/grakn.nix { };
```

Please sort the package in the right section alphabetically.